main/CommonPrelude.c

```diff
@@ -159,4 +159,28 @@ const char ctagsCommonPrelude []=
 "    } forall\n"
 "    pop\n"
 "} __bddef\n"
+"\n"
+"(string end:string _ENDWITH boolean)\n"
+"/_endwith {\n"
```

Comment:
`_strendwith` may be better name.

Reply:
Using _strrstr is one of the ways to shorten the proc.
Writing in C is another choice.